ui/src/app/base/components/ColumnToggle/_index.scss

```diff
@@ -1,22 +1,16 @@
 @mixin ColumnToggle {
-  td:not(.p-table__expanding-panel) [class*="p-button"].column-toggle,
   .column-toggle {
     align-items: center;
     color: $color-dark;
-    display: flex;
-    height: 100%;
-    justify-content: space-between;
+    text-align: left;
     width: 100%;
+  }
 
-    &::after {
-      @include vf-icon-chevron;
-      @include vf-icon-size($default-icon-size);
-      content: "";
-    }
+  .column-toggle__label {
```

Comment:
This could use the `u-flex--grow` class if you want to remove one more custom class.

Reply:
Forgot about those, thanks!